pkg/yang/identity_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package yang
+
+import "testing"
+
+type modIn struct {
+	name    string
+	content string
+}
+
+type modOut struct {
+	name       string
+	identities []*idOut
+}
+
+type idOut struct {
+	idName   string
+	baseName string
+}
+
+type testCase struct {
+	name string
+	in   []*modIn
```

Comment:
Per the comments from @andaru via email, I modified all references within the `identity_test` to make them directly include the relevant entity rather than reference it.